app/assets/stylesheets/application.scss

```diff
@@ -88,3 +88,30 @@ input.input-field-error {
 #save-button {
   margin-top: 2em;
 }
+
+.border-form-div {
+  max-width: 300px;
+  padding: 19px 29px 29px;
+  margin: 0 auto 20px;
+  background-color: #fff;
```

Comment:
This looks great!  Can we update the background color to be #e9e9e9 to match the grey background of the registration and edit profile pages?